projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-following-recommendations.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * REST API endpoint to retrieve following recommendations
+ *
+ * @package automattic/jetpack
+ * @since $$next-version$$
+ */
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Status\Visitor;
+
+/**
+ * Class WPCOM_REST_API_V2_Endpoint_Following_Recommendations
+ */
+class WPCOM_REST_API_V2_Endpoint_Following_Recommendations extends WP_REST_Controller {
+	/**
+	 * Namespace prefix.
+	 *
+	 * @var string
+	 */
+	public $namespace = 'wpcom/v2';
+
+	/**
+	 * Endpoint base route.
+	 *
+	 * @var string
+	 */
+	public $rest_base = 'following';
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->wpcom_is_wpcom_only_endpoint    = true;
+		$this->wpcom_is_site_specific_endpoint = false;
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/recommendations',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_response' ),
+					'permission_callback' => 'is_user_logged_in',
+				),
+			)
+		);
+	}
+
+	/**
+	 * Gets the sites the user is following
+	 *
+	 * @return array|WP_Error list of followed sites, WP_Error otherwise
+	 */
+	public function get_response() {
+		$body = Client::wpcom_json_api_request_as_user(
```

Comment:
Do we have a way to shortcircuit this on WordPress.com simple sites, where we may be able to query for the information without having to make that API request? You can check `projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-ai.php` for an example of this.